src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -417,12 +552,26 @@ TEST(hlpTimestampTests, specific_format)
     ASSERT_EQ("5", result["_stampTs.seconds"]);
 }
 
-TEST(domain_test, success)
+TEST(hlpTests_Timestamp, kitchen)
```

Comment:
As this test is known to fail, we should add a comment explaining this

Reply:
done